bbb-export-annotations/workers/process.js

```diff
@@ -291,14 +291,18 @@ function overlayAnnotations(svg, slideAnnotations) {
  * @param {number} page Page number.
  */
 function resizeAssetIfBase64Image(file, width, height, oldWidth, oldHeight, page) {
-  const BASE_64_HREF_REGEX = /href="data:image\/(png|jpg|jpeg);base64,[^"]+"/;
+  const BASE_64_HREF_REGEX = /href="data:image\/(png|jpg|jpeg);base64,[^"]+"/g;
   const isSvg = file.endsWith('.svg');
 
   if (!isSvg) return;
 
   try {
     let svg = fs.readFileSync(file, { encoding: 'utf-8' });
-    const hasBase64Image = BASE_64_HREF_REGEX.test(svg);
+    let count = 0;
+    while (BASE_64_HREF_REGEX.exec(svg) !== null) {
+      count++;
+    }
+    const hasBase64Image = count === 1;
```

Comment:
I'm wondering what will happen when count > 1?
Will the bug fixed here come back? https://github.com/bigbluebutton/bigbluebutton/pull/22807

Reply:
`count === 1` is the only case we need to tackle. I retested #22807 and couldn't spot any regressions.

Reply:
An inline comment on why 1 is special here (and what would it mean to be <1 or >1 or === 1) would be greatly appreciated :pray: